README.md

```diff
@@ -47,9 +47,9 @@ KubeEdge is composed of the following components:
 
 ## Usage
 
-* [Manual Installation of Docker, Kubernetes, Mosquitto](./docs/getting-started/usage.md#prerequisites)
-* [One click Kubeedge Installer to install both Cloud and Edge nodes](./keadm/README.md)
-* [Run KubeEdge](./docs/getting-started/usage.md#run-kubeedge)
+* [One click KubeEdge Installer to install both Cloud and Edge nodes](./keadm/README.md)
+* [Run KubeEdge from release package](./docs/getting-started/release_package.md)
+* [Run KubeEdge from source](./docs/getting-started/usage.md#prerequisites)
```

Comment:
why is this link pointing to prerequisites ?

Reply:
will remove..